ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/router-bgp.md

```diff
@@ -940,6 +944,10 @@
           bgp_listen_range_prefix: <str>
           session_tracker: <str>
 
+          # To apply the profile name, it must be created under "management-security".
+          shared_secret_profile_name: <str>
```

Comment:
Tried to apply config via eapi but getting error:
```

TASK [arista.avd.eos_config_deploy_eapi : Replace configuration with intended configuration] *****************************************************
fatal: [leaf1-dc1]: FAILED! => {"changed": false, "module_stderr": "Invalid input (at token 7: 'hmac-sha-256')", "module_stdout": "", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error"}
```

Reply:
Tested with CVP and it is working.

Reply:
It is working ok with EOS version 4.31 and above. We can have a note in description